client/installer/list_clusters_parameters.go

```diff
@@ -73,6 +73,11 @@ for the list clusters operation typically these are written to a http.Request
 */
 type ListClustersParams struct {
 
+	/*AmsSubscriptionIds
+	  An array of cluster to retrieve from their subscription id.
```

Comment:
Suggest something like: "If non-empty, returned Clusters are filtered to those with a matching subscription ID."

Reply:
That indeed sounds better.
I will change it, thanks.